miasm2/jitter/jitload.py

```diff
@@ -310,14 +328,16 @@ def runiter_once(self, pc):
 
         # Check breakpoints
         old_pc = self.pc
-        for res in self.breakpoints_handler.call_callbacks(self.pc, self):
-            if res is not True:
-                if isinstance(res, collections.Iterator):
+        callback = self.breakpoints_handlers.get(self.pc, None)
+        if callback:
```

Comment:
An explicit pointer test against `None` is faster to detect the dict miss (avoid testing if a function is equal to 0)